timeless/restaurants/reservations/views.py

```diff
@@ -0,0 +1,46 @@
+"""Reservations views module.
+
+"""
+
+
+from flask import (
+    Blueprint, flash, redirect, render_template, request, url_for
+)
+
+
+bp = Blueprint("reservations", __name__, url_prefix="/reservations")
+
+
+@bp.route("/")
+def list_reservations(reservations):
```

Comment:
@paulodamaso Add pls puzzle to refactor this when GenericViews will be implemented
One of 
https://github.com/timelesslounge/timelessis/issues/173
https://github.com/timelesslounge/timelessis/issues/134